Volume/Networking/Network.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Network.swift
+//  Volume
+//
+//  Created by Daniel Vebman on 11/22/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Apollo
+import Combine
+import Foundation
+
+/*
+ // Example
+ @State private var launches: [LaunchListQuery.Data.Launch.Launch] = []
+ @State private var cancellableQuery: AnyCancellable?
+ @State private var didCancel = false
+ 
+ func fetch() {
+     didCancel = false
+     cancellableQuery = Network.shared.apollo.fetch(query: LaunchListQuery())
+         .map { data in data.launches.launches.compactMap { $0 } }
+         .sink(receiveCompletion: { completion in
+             if case let .failure(error) = completion {
+                 print(error.localizedDescription)
+             }
+         }, receiveValue: { value in
+             launches = value
+         })
+ }
+ */
+
+class Network {
+    static let shared = Network()
+    
+    // TODO: change endpoint
+    // Use #if DEBUG to differentiate staging/dev from prod
+    let apollo = ApolloClient(url: URL(string: "http://volume-backend.cornellappdev.com/graphql")!)
```

Comment:
if this is the actual backend endpoint, this is something we definitely don't want to expose -- i.e. drop this commit/rebase so it doesn't even show up in the commit history. instead, we'd want a Keys.swift and a Keys.plist (that doesn't get committed)   -- i can help out with the rebasing if you're not comfortable with doing it!